### DISCUSSION TITLE
Octernship Submissions

### DISCUSSION THREAD

**Discussion by chaitak-gorai** [community/community#51100] [GitHub Education]
I applied for github octernships and got assigned with some task in the repository,
I have completed the task but can't figure out how to submit it

The due date is by 2nd April will it automatically be considered after 2nd or do i need to submit anything.

Please clear these doubt anyone who have submitted their task.

Thank you

**Comment by Ntshangase** [comment]
Hello,

It's recommended that you contact the mentor for that project for clarification on how to submit.

**Reply by gopalkumr** [reply]
How can I contact the appwrite mentor for our project ?

**Reply by ashwinexe** [reply]
> How can I contact the appwrite mentor for our project ?

Appwrite discord :)

**Comment by sumitshinde-84** [comment]
yes you are absolutely right @Ntshangase ,
because every organization accept task solution by different different way so its better to contact to your mentor ,
and if you get touch with your mentor it will be benificial for you further .
so try to contact metor as early as possible .
@chaitak-gorai

**Comment by chaitak-gorai** [comment]
Thank you @Ntshangase @sumitshinde-84 :)

**Reply by Ntshangase** [reply]
Don't  forget to "Mark as answer", if you found the response helpful.

**Comment by ashwinexe** [comment]
Your task is auto-submitted to organization after the deadline. Make sure you have your final PR merged to main so maintainers can see what changes you made :)

**Reply by chaitak-gorai** [reply]
Thank you

**Comment by PK284** [comment]
In Textify project submission it is mentioned that the Students are expected to submit their work as either presentations/ video submissions/ live product demos, etc. Using the auto-generated Feedback Pull Request for review and submission Using GitHub Discussions to ask any relevant questions regarding the project
But I am unable to find any Auto-generated pull request in my repository.
Kindly help and do tell me which moderator I need to contact for my submission.
![image](https://user-images.githubusercontent.com/72080733/229065474-de54b982-2fde-403b-a45f-0e73d795c089.png)
